administration/src/application/components/ApplyController.tsx

```diff
@@ -39,12 +41,12 @@ const ApplyController = () => {
       return
     }
 
-    const regionId = 1 // TODO: Add a mechanism to retrieve the regionId
+    const [regionId, applicationInput] = application.value // TODO: Add a mechanism to retrieve the regionId
```

Comment:
Is this TODO sill up to date? The regionId is come now from somewhere it seems.

Reply:
Yeah, it's still valid.